packages/venia-concept/src/actions/app/__tests__/asyncActions.spec.js

```diff
@@ -0,0 +1,75 @@
+import { store } from 'src';
+import actions from '../actions';
+import { closeDrawer, loadReducers, toggleDrawer } from '../asyncActions';
+
+// `src/index.js` performs side effects and depends on magic webpack globals
+// all we need is the store it exports, so we mock it
+jest.mock('src');
```

Comment:
Ah, I see we are using the index page for its store. I think this shows us we need to refactor the index page, so that it imports a different file which creates the store from a pure function. That would enable testing without so much mocking!

(And when doing a PR that is "just to add test coverage", it's totally fine to refactor code to make it more testable.)